app/presenters/bill-runs/index-bill-runs.presenter.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+/**
+ * Formats the summary data for each bill run for use in the /bill-runs page
+ * @module IndexBillRunsPresenter
+ */
+
+const {
+  capitalize,
+  formatBillRunType,
+  formatLongDate,
+  formatMoney
+} = require('../base.presenter.js')
+
+/**
+ * Formats the summary data for each bill run for use in the /bill-runs page
+ *
+ * @param {module:BillRunModel[]} billRuns - The bill runs containing the data to be summarised for the view
+ *
+ * @returns {Object[]} Each bill run summary formatted for use in the `index.njk` template for `/bill-runs`
+ */
+function go (billRuns) {
+  return billRuns.map((billRun) => {
+    const {
+      batchType,
+      billRunNumber,
+      createdAt,
+      id,
+      netTotal,
+      numberOfBills,
+      region,
+      scheme,
+      summer,
+      status
+    } = billRun
+
+    const type = formatBillRunType(batchType, scheme, summer)
```

Comment:
Just curious what made you choose to define type here and not inside the returned object like what you have done with createdAt, link, region and total?

Reply:
I think it comes from a refactor. I previously needed `type` for something else but now no longer do and have overlooked updating the code. Good spot!